src/Arcus.Observability.Tests.Unit/Telemetry/LogEventExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Linq;
+using GuardNet;
+
+// ReSharper disable once CheckNamespace
+namespace Serilog.Events
```

Comment:
Move this in testing or our core?

Reply:
Ah, can you think of scenario's where this would come in handy maybe? Then yeah, sure!

Reply:
Yes, in enrichers and so

Reply:
But ok to keep for now if you prefer

Reply:
I'll move it.